### PR TITLE
Four Cybersecurity Attacks Your Employees Need to Know

### PR DESCRIPTION
**Pull request by dan-auth0**

### CODE REVIEW

File: _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Still, there are ways to protect yourself. Here are four of the most common cyberattack examples and the ways you can avoid them.
+
+## **1. Malware Infiltration**
+
+**What makes you vulnerable: outdated software and lack of backup protocols**
+
+Malware is a term that encompasses many different types of malicious software, including ransomware and spyware. Each type of malware can wreak havoc once it gains access to a system.
+
+WannaCry, for example, a form of ransomware, recently [took advantage](https://www.symantec.com/security-center/writeup/2017-051310-3522-99) of a weak spot in Microsoft’s operating system. Banks, health care providers, manufacturers, and other businesses across the globe reported WannaCry's encryption.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown]: `...[took advantage](https://www.symantec.com/security-center/writeup/2017-051310-3522-99)...`

"...[took advantage of a weak spot in Microsoft’s operating system](https://www.symantec.com/security-center/writeup/2017-051310-3522-99)..."

File: _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+_Solution: Regularly updating your systems will go a long way toward removing dangerous holes. It's a simple fix that many teams, including [Equifax](https://www.wired.com/story/equifax-breach-no-excuse/), have neglected._
+
+_If you want added security, consider [moving to the cloud](https://auth0.com/blog/moving-your-app-to-cloud/). Not only will this allow your enterprise more flexibility to scale, but you can also delegate specialized security tasks like IAM and encryption to expert providers._
+
+## **2. Phishing **
+
+**What makes you vulnerable: lack of education and attention to detail in suspicious e-mails**
+
+In 2016, [91% of cyberattacks](https://phishme.com/2016-enterprise-phishing-susceptibility-report/)were the result of a spear-phishing email. Phishing e-mails steal data either by tricking recipients into giving it to the cyber criminals or by getting users to download a file that installs spyware—software designed to steal information directly from a user’s device.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown]: `In 2016, [91% of cyberattacks](https://phishme.com/2016-enterprise-phishing-susceptibility-report/)were the...`

Missing space before *were*.

File: _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+Children’s Mercy Hospital [_recently reported a breach_](https://healthitsecurity.com/news/childrens-mercy-hospital-phishing-attack-impacts-phi-of-60k) that exposed the health care information of at least 60,000 people. E-mails tricked users into disclosing their work e-mail credentials, which led to the exposure of patient health data when the hackers gained access to the inboxes of the targeted hospital workers. Children’s Mercy [_is now being sued_](https://www.hipaajournal.com/childrens-mercy-hospital-sued-for-63000-record-data-breach/) for this major breach.
+
+Here’s an example of an imitation Dropbox e-mail asking for confirmation of a user’s e-mail:
+
+<p style="text-align: center;">
+  <img src="https://cdn.auth0.com/blog/four-cybersecurity-attacks-you-need-to-know/dropbox-email-asking-confirmation-of-user-email.png" alt="Dropbox e-mail asking for confirmation of a user’s e-mail">
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown]: `Dropbox e-mail asking for confirmation of a user’s e-mail`

"Imitation Dropbox email phishing scam asking for confirmation of a user’s e-mail"

File: _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+_Solution: As these e-mails start to look more realistic, it’s important to educate employees on how to spot a questionable e-mail chain._
+
+_Creative training programs include the following:_
+
+- _Digital Defense's [SecurED training program](https://www.digitaldefense.com/professional-services/secured-training/) that works with award-winning Hollywood comedy writers_
+- _Terranova's [customizable newsletters](https://terranovacorporation.com/solution/communication-material/)_
+- _Inspired eLearning's [Security Awareness for the C-Suite](https://inspiredelearning.com/security-awareness/awareness-c-suite/)_
+- _Wombat Security's [everyday tips](https://www.wombatsecurity.com/security-education/free-security-awareness-resources)_
+
+_See [here](https://www.csoonline.com/article/3225471/security/please-dont-send-me-to-cybersecurity-training.html) for a larger list._
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown]: `_See [here](https://www.csoonline.com/article/3225471/security/please-dont-send-me-to-cybersecurity-training.html) for a larger list._`

"_[See here for a larger list](https://www.csoonline.com/article/3225471/security/please-dont-send-me-to-cybersecurity-training.html)._"

File: _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+_Solution: Outlawing anonymous, external drives on company devices will send a clear message to your organization. Whether a drive was planted or simply purchased from an illegitimate source, removing the temptation to plug something in without properly vetting is a low-hanging fruit that can have enormous power to secure your team and your end users._
+
+## **4. Mobile Attacks**
+
+**What makes you vulnerable: employees accessing company data and networks via cell phones**
+
+Hackers are well aware of the explosion of smartphone and tablet use and have created attacks specifically designed for mobile software. [_The Pegasus attack_](https://blog.lookout.com/lookout-trident-pegasus-enterprise-discovery) on Apple’s iOS software is a prime example. Pegasus infected iPhones through phishing text messages that asked recipients to click on a link inside the text message.
+
+<p style="text-align: center;">
+  <img style="width: 300px;" src="https://cdn.auth0.com/blog/four-cybersecurity-attacks-you-need-to-know/pegasus-mobile-attack-on-ios.png" alt=" _The Pegasus attack_ (https://blog.lookout.com/lookout-trident-pegasus-enterprise-discovery) on Apple’s iOS software">
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown]: ` _The Pegasus attack_ (https://blog.lookout.com/lookout-trident-pegasus-enterprise-discovery) on Apple’s iOS software`

"Example of Pegasus attack on Apple iOS mobile software"

File: _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+_Solution: If a mobile phone or tablet has been breached, you need a Plan B. Make sure employees quickly change their passwords by using a professional password manager, such as [LastPass](https://www.lastpass.com/) or [Valt](https://valt.io/). Many employees secure critical information with easy-to-guess iterations of a single password. Encouraging length and complexity in all passwords going forward will make it difficult for hackers to access multiple areas at once._
+
+## Additional Security for All Systems at the Login Stage
+
+Good password policy and sound employee education are key; however, taking the next step with [secure login procedures](https://auth0.com/b2c-customer-identity-management) will add a new dimension of safety to your system. Specialized identity solutions providers can offer support with [multifactor authentication](https://auth0.com/docs/multifactor-authentication) and [passwordless](https://auth0.com/docs/connections/passwordless) registration.
```
Comment (scottmathson-auth0) [inline _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown]: Change passwordless link to: https://auth0.com/passwordless

Comment (dan-auth0) [reply _posts/2018-07-30-four-cybersecurity-attacks-you-need-to-know.markdown]: @scottmathson-auth0 Unable to update. Content on link needs update.

**Pull request merged**